### PR TITLE
Implement update_element/3

### PR DESCRIPTION
**Pull request by APB9785**
Wraps :ets.update_element/3

### CODE REVIEW

File: test/ets/set_test.exs [hunk 1]
```diff
@@ -563,7 +563,7 @@ defmodule SetTest do
   end
 
   describe "Delete" do
-    test "delete!/2 raises on error" do
+    test "delete!/1 raises on error" do
```
Comment (APB9785) [inline test/ets/set_test.exs:629]: Unrelated but I noticed these arities were swapped

File: test/ets/set_test.exs [hunk 2]
```diff
@@ -572,7 +572,7 @@ defmodule SetTest do
       end
     end
 
-    test "delete!/1 raises on error" do
+    test "delete!/2 raises on error" do
```
Comment (APB9785) [inline test/ets/set_test.exs:638]: Unrelated but I noticed these arities were swapped

**Comment by TheFirstAvenger** [comment]
🎖❤️⭐️⚪️⭐️💙🎖
Thanks!

**Comment by TheFirstAvenger** [comment]
Available now in v0.9.0

**Pull request merged**